packages/twenty-front/src/hooks/useSortedArray.ts

```diff
@@ -0,0 +1,39 @@
+import { useRecoilValue } from 'recoil';
+
+import { tableSortFamilyState } from '@/activities/states/tableSortFamilyState';
+import { OrderBy } from '@/object-metadata/types/OrderBy';
+
+export const useSortedArray = <T>(
+  arrayToSort: T[],
+  { tableId, initialFieldName }: { tableId: string; initialFieldName: string },
+): T[] => {
+  const sortConfig = useRecoilValue(
+    tableSortFamilyState({ tableId, initialFieldName }),
+  );
+
+  const sortTableData = (
+    data: T[],
```

Comment:
Please avoid variable names like `data` here it should be tableRows or something more contextualized